### PR TITLE
Closes #2930 - When First Run is dismissed, URL bar appears in purple typing state

### LINKED ISSUE #2930: When First Run is dismissed, URL bar appears in purple typing state
## Steps to reproduce
1. Dismiss First Run

### Expected behavior
URL Bar in normal active (pink) state

### Actual behavior
URL Bar in purple typing state 

Probably a regression from #2643 .

### PR DESCRIPTION
**Pull request by ShridharGoel**
**Earlier:**

After dismissing First Run, URL bar appeared in purple typing state.

**Now:**

After dismissing First Run, URL bar will appear in normal active (pink) state.

### CODE REVIEW

**Comment by ShridharGoel** [comment]
Any idea why the Taskcluster check is failing?

**Comment by ekager** [comment]
Hi @ShridharGoel ! Looks like it failed detektCheck with error:
 MaxLineLength - [UrlInputFragment.kt] at focus-android/app/src/main/java/org/mozilla/focus/fragment/UrlInputFragment.kt:252:124

File: app/src/main/java/org/mozilla/focus/fragment/UrlInputFragment.kt [hunk 1]
```diff
@@ -244,8 +245,12 @@ class UrlInputFragment :
 
         context?.let {
             if (!Settings.getInstance(it).shouldShowFirstrun()) {
+
                 // Only show keyboard if we are not displaying the first run tour on top.
                 showKeyboard()
+
+                @Suppress("MaxLineLength")
+                toolbarBackgroundView.background = ContextCompat.getDrawable(context!!, R.drawable.animated_background_url)
```
Comment (pocmo) [inline app/src/main/java/org/mozilla/focus/fragment/UrlInputFragment.kt]: Please just use a line break instead of suppressing this here 😅

Comment (ShridharGoel) [reply app/src/main/java/org/mozilla/focus/fragment/UrlInputFragment.kt]: Done.

**Pull request merged**